frontend/src/worksheets/FilterWorkSheets.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+
+function FilterWorkSheets(props) {
+  const { onStatusChange, status } = props
+
+  const handleInputChange = (event) => {
+    onStatusChange(event.target.value)
+  }
+
+  return (
+    <div>
+      <form>
+        <select
+          value={status}
+          class="btn btn-success"
+          onChange={handleInputChange}
+          name="filterWorkSheetByStatus"
+        >
+          <option value="CREATED" label="Nyitott">
+            Nyitott
+          </option>
+          <option value="REPORTED" label="Készre jelentett">
+            Készre jelentett
+          </option>
+          <option value="CLOSED" label="Lezárt">
+            Lezárt
+          </option>
+          <option value="" label="Mind">
+            Lezárt
```

Comment:
A mind is lezárt szöveget ír ki?

Reply:
Illetve szerintem vagy a labelt használjuk vagy a childrent mindenhol, a kettő együtt kicsit redundáns.

Reply:
Labeleket töröltem. Valóban indokolatlanul voltak benne.